mk/config.mk

```diff
@@ -170,3 +170,8 @@ CFG_SMALL_PAGE_USER_TA ?= y
 # Enable support for detected undefined behavior in C
 # Uses a log of memory, can't be enabled by default
 CFG_CORE_SANITIZE_UNDEFINED ?= n
+
+# Device Tree support
+# When enabled, the TEE _start function expects to find the address of a
+# Device Tree Blob (DTB) in register r2. The DT parsing code relies on libfdt.
```

Comment:
+ say what happens currently when it's enabled? (i.e., adds optee and reserved-memory nodes). One might think it's used to discover devices.